.github/workflows/inspektor-gadget.yml

```diff
@@ -227,6 +239,8 @@ jobs:
           -czf ${client}.tar.gz -C $platform \
           ${{ matrix.client }}${dot_exe} LICENSE
         rm -rf $platform
+    - name: CVE checks for ${{ matrix.client }}-${{ matrix.os }}-${{ matrix.arch }}
```

Comment:
I'm a bit confused by CVE being mentioned there, it seems to me it's about code scanning rather than just CVEs, right?